build/templates/rep_caps.rst.mako

```diff
@@ -40,27 +44,43 @@ ${helper.get_rst_header_snippet(name, '-')}
     .. py:attribute:: ${module_name}.Session.${name}[]
 
 % if len(prefix) > 0:
-        If no prefix is added to the items in the parameter, the correct prefix will be added when
-        the driver function call is made.
+        ..
+            If no prefix is added to the items in the parameter, the correct prefix will be added when
+            the driver function call is made.
 
-        .. code:: python
+            .. code:: python
 
-            session.${name}['0-2'].channel_enabled = True
+                session.${name}['0-2'].channel_enabled = True
 
-        passes a string of :python:`'${prefix}0, ${prefix}1, ${prefix}2'` to the set attribute function.
+            passes a string of :python:`'${prefix}0, ${prefix}1, ${prefix}2'` to the set attribute function.
 
-        If an invalid repeated capability is passed to the driver, the driver will return an error.
+            If an invalid repeated capability is passed to the driver, the driver will return an error.
```

Comment:
This is usually implied and can be omitted. For method parameters, we don't say something like this explicitly.